core/src/main/res/values-fr/strings.xml

```diff
@@ -254,26 +254,26 @@
   <string name="nearby_devices">APPAREILS À PROXIMITÉ</string>
   <string name="no_devices_found">Aucun appareil détecté. Tapez le bouton Rechercher pour essayer à nouveau.</string>
   <string name="files_for_transfer">FICHIERS À TRANSFÉRER</string>
-  <string name="preparing_files">Préparation des fichiers pour le transfert…</string>
-  <string name="performing_handshake">Établissement de liaison en cours…</string>
+  <string name="preparing_files">Préparation des fichiers pour le transfert...</string>
+  <string name="performing_handshake">Établissement de liaison en cours...</string>
```

Comment:
We should revert these changes.

Reply:
I have fixed the problem upstream in translatewiki. Next update should be good.